numpy/core/overrides.py

```diff
@@ -66,29 +66,32 @@
     """)
 
 
-ArgSpec = collections.namedtuple('ArgSpec', 'args varargs keywords defaults')
-
-
 def verify_matching_signatures(implementation, dispatcher):
```

Comment:
Since this function is a startup cost, I think it would pay to optimize as
```
_Parameter_empty = inspect.Parameter.empty
```
and use `_Parameter_empty` within the function body

Reply:
Good idea, applied in 6e35c2a